azdev/operations/code_gen.py

```diff
@@ -55,24 +60,22 @@ def create_module(mod_name='test', display_name=None, display_name_plural=None,
     _display_success_message(COMMAND_MODULE_PREFIX + mod_name, mod_name)
 
 
-def create_extension(ext_name='test', repo_name='azure-cli-extensions',
-                     display_name=None, display_name_plural=None,
-                     required_sdk=None, client_name=None, operation_name=None, sdk_property=None,
-                     not_preview=False, github_alias=None, local_sdk=None):
+def create_extension(ext_name, azure_rest_api_specs=const.GITHUB_SWAGGER_REPO_URL, use=None):
+    require_virtual_env()
+    repo_name = const.EXT_REPO_NAME
```

Comment:
if repo_name is just set to a constant and never changed, just use the const.EXT_REPO_NAME in place of repo_name